code/game/machinery/computer/card.dm

```diff
@@ -320,6 +324,7 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 				id_insert_modify(usr)
 		if ("inserted_scan_id")
 			if (inserted_scan_id)
+				authenticated = LOGGED_OFF

```

Comment:
Wouldn't this make more sense to be in id_eject_scan() itself? Or am I way off base on that?

Reply:
You're right, it was already done in id_eject_scan(). Removed this unnecessary one.